ably/realtime/connection.py

```diff
@@ -55,7 +56,9 @@ async def close(self):
         self.__state = ConnectionState.CLOSED
 
     async def connect_impl(self):
-        async with websockets.connect(f'wss://{self.options.realtime_host}?key={self.ably.key}') as websocket:
+        headers = HttpUtils.default_get_headers()
```

Comment:
I'm actually not sure if we wan't to be sending the `Accept` header with these requests or not. As far as I can tell the other realtime SDKs don't send this header so I would prefer not to do so here.

I think you should create a new static `HttpUtils.default_headers()` method which just gives you the `X-Ably-Version` and `Ably-Agent` headers and use that (and also refactor the `default_get_headers()` to use it).